blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -111,11 +108,11 @@ public void startServer() {
         Channel serverChannel = server.bind(new InetSocketAddress(httpIngestHost, httpIngestPort));
         allOpenChannels.add(serverChannel);
 
-        log.info("Starting tracker service");
-        tracker = new Tracker();
+        //register the tracker MBean for JMX/jolokia
+        log.info("Registering tracker service");
+        Tracker.getInstance();
```

Comment:
Note that if I don't have this call here in the ingest/queryServer.startServer() method, the track(), trackResponse(), and trackDelayedMetricsTenant() in the handlers still work (all code functionalities within blueflood will still work as intended), but we will not be able to see the MBean in JMX/jolokia on service startup because it hasn't been registered yet, and will not be able to make any of the calls to the MBean (such as addTenant, addMetricName, setIsTrackingDelayedMetrics) until at least one of the call is made in the ingest or query handlers (at which time the registration is made from the private constructor).

Doing it here in startServer() register the MBean preemptively.